app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -7,7 +7,7 @@
   <%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
 <% end %>
 <%= presenter.attribute_to_html(:keyword, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:based_near_label, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:based_near_label) %>
```

Comment:
rendering the text as faceted resulted in an empty search so I removed the facet.